src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml.cpp

```diff
@@ -0,0 +1,282 @@
+                                                                              : m_resourceLoader->GetString(L"KGFHorizontalAsymptotesNone");
+            ObliqueAsymptotes = ViewModel->ObliqueAsymptotes->Size != 0 ? ConvertVectorToString(ViewModel->ObliqueAsymptotes)
+                                                                        : m_resourceLoader->GetString(L"KGFObliqueAsymptotesNone");
+            SetMonotoncityStringProperty();
+            SetTooComplexFeaturesErrorProperty();
+        }
+    }
+}
+
+void KeyGraphFeaturesPanel::SetParityStringProperty()
+{
+    if (ViewModel->Parity)
+    {
+        switch (ViewModel->Parity)
+        {
+        case 0:
+            Parity = m_resourceLoader->GetString(L"KGFParityUnknown");
+            break;
+        case 1:
+            Parity = m_resourceLoader->GetString(L"KGFParityOdd");
+            break;
+        case 2:
+            Parity = m_resourceLoader->GetString(L"KGFParityEven");
+            break;
+        case 3:
+            Parity = m_resourceLoader->GetString(L"KGFParityNeither");
+            break;
+        default:
+            Parity = m_resourceLoader->GetString(L"KGFParityError");
+        }
+    }
+}
+
+void KeyGraphFeaturesPanel::SetPeriodicityStringProperty()
+{
+    if (ViewModel->Periodicity->Size != 0)
+    {
+        auto periodicityType = ViewModel->Periodicity->First()->Current->Key->Data();
+        switch (*periodicityType)
+        {
+        case '0':
+            Periodicity = m_resourceLoader->GetString(L"KGFPeriodicityUnknown");
+            break;
+        case '1':
+            Periodicity = ViewModel->Periodicity->First()->Current->Value;
+            break;
+        case '2':
+            Periodicity = m_resourceLoader->GetString(L"KGFPeriodicityNotPeriodic");
+            break;
+        default:
+            Periodicity = m_resourceLoader->GetString(L"KGFPeriodicityError");
+        }
+    }
+}
+
+void KeyGraphFeaturesPanel::SetMonotoncityStringProperty()
+{
+    if (ViewModel->Monotonicity->Size != 0)
+    {
+        int monotonicitySize = ViewModel->Monotonicity->Size;
+        int i = 0;
+        for (auto item : ViewModel->Monotonicity)
+        {
+            Monotonicity += item->Key;
+            Monotonicity += " ";
+
+            auto monotonicityType = item->Value->Data();
+            switch (*monotonicityType)
+            {
+            case '0':
+                Monotonicity += m_resourceLoader->GetString(L"KGFMonotonicityUnknown");
+                break;
+            case '1':
+                Monotonicity += m_resourceLoader->GetString(L"KGFMonotonicityIncreasing");
+                break;
+            case '2':
+                Monotonicity += m_resourceLoader->GetString(L"KGFMonotonicityDecreasing");
+                break;
+            case '3':
+                Monotonicity += m_resourceLoader->GetString(L"KGFMonotonicityConstant");
```

Comment:
This is basically building in a string that has 2 parts. This first is a mathml string, which I cannot localize and the second is just a direction. Output looks like [0, 1] increasing. However, it could also be Increasing [0, 1], so I don't think we need to change the order in this case.